### PR TITLE
housekeeping: find and replace nventive/uno to unoplatform/uno

### PR DESCRIPTION
**Pull request by ghuntley**
## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## What is the current behavior?

References to nventive GitHub org

## What is the new behavior?

Links to correct github org

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

**Comment by XiaotianNetlift** [comment]
There are stilla few more in the source code:
```
D:\code\framework\uno\src\SolutionTemplate\UnoSolutionTemplate\UWP\UnoQuickStart.Uwp.csproj
  9,91:	This is not an issue when libraries are referenced through nuget packages. See https://github.com/nventive/Uno/issues/446 for more details.

D:\code\framework\uno\src\Uno.UI.BindingHelper.Android\Uno\UI\UnoViewGroup.java
  432,76:	// Note: this is not fully complient with the UWP contract (cf. https://github.com/nventive/Uno/issues/649)

D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.xaml
  240,54:	<!-- Based on is required until https://github.com/nventive/Uno/issues/119 is fixed -->
  424,70:	- Canvas.ZIndex is not supported in Uno Grid yet, see https://github.com/nventive/Uno/issues/325

D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\Primitives\CarouselPanel.cs
  10,84:	public partial class CarouselPanel : StackPanel //TODO: VirtualizingPanel https://github.com/nventive/Uno/issues/1133

D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\TextBox\TextBox.cs
  679,21:	// TODO (https://github.com/nventive/Uno/issues/683): && ActualWidth >= TDB / Note: We also have to invoke this method on SizeChanged

D:\code\framework\uno\src\Uno.UI\UI\Xaml\FrameworkElement.cs
  61,19:	/// See https://github.com/nventive/Uno/issues/119 for details.

D:\code\framework\uno\src\Uno.UI\UI\Xaml\Media\Animation\DoubleAnimation.cs
  353,22:	//		 cf. https://github.com/nventive/Uno/issues/631
```
Do we want to replace them as well?

File: doc/blog/201809-UnoUnderTheHoodXamlGeneration.md [hunk 1]
```diff
@@ -82,4 +82,4 @@ In the normal course of Uno development you don't need to look at code generated
...
+One last fun fact: when building on MacOS using [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/) (which Uno tentatively supports in preview), Uno uses an internal port in place of the System.Xaml namespace which isn't available on Mac. The same code backs Uno's support for runtime Xaml interpretation via [Windows.UI.Xaml.Markup.XamlReader](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.markup.xamlreader). That in turn is used in the [interactive mode](https://github.com/unoplatform/Uno.Playground/blob/master/src/Uno.Playground.Shared/Samples/Playground.xaml) of the [Uno Gallery app](https://github.com/unoplatform/Uno.Playground) and [Uno.Playground website](https://playground.platform.uno/), which allows you to edit snippets of XAML and see the results in realtime. Check it out!
```
Comment (carldebilly) [inline doc/blog/201809-UnoUnderTheHoodXamlGeneration.md]: Is it ok to change it in published blog posts?

Comment (ghuntley) [reply doc/blog/201809-UnoUnderTheHoodXamlGeneration.md]: Yup.

**Comment by ghuntley** [comment]
> 
> 
> There are stilla few more in the source code:
> 
> ```
> D:\code\framework\uno\src\SolutionTemplate\UnoSolutionTemplate\UWP\UnoQuickStart.Uwp.csproj
>   9,91:	This is not an issue when libraries are referenced through nuget packages. See https://github.com/nventive/Uno/issues/446 for more details.
> 
> D:\code\framework\uno\src\Uno.UI.BindingHelper.Android\Uno\UI\UnoViewGroup.java
>   432,76:	// Note: this is not fully complient with the UWP contract (cf. https://github.com/nventive/Uno/issues/649)
> 
> D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\NavigationView\NavigationView.xaml
>   240,54:	
>   424,70:	- Canvas.ZIndex is not supported in Uno Grid yet, see https://github.com/nventive/Uno/issues/325
> 
> D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\Primitives\CarouselPanel.cs
>   10,84:	public partial class CarouselPanel : StackPanel //TODO: VirtualizingPanel https://github.com/nventive/Uno/issues/1133
> 
> D:\code\framework\uno\src\Uno.UI\UI\Xaml\Controls\TextBox\TextBox.cs
>   679,21:	// TODO (https://github.com/nventive/Uno/issues/683): && ActualWidth >= TDB / Note: We also have to invoke this method on SizeChanged
> 
> D:\code\framework\uno\src\Uno.UI\UI\Xaml\FrameworkElement.cs
>   61,19:	/// See https://github.com/nventive/Uno/issues/119 for details.
> 
> D:\code\framework\uno\src\Uno.UI\UI\Xaml\Media\Animation\DoubleAnimation.cs
>   353,22:	//		 cf. https://github.com/nventive/Uno/issues/631
> ```
> 
> Do we want to replace them as well?

Good catch - fixed.

**Comment by ghuntley** [comment]
> 
> 
> There are a few deadlinks where the repo is still on the `/nventive` side:
> [unoplatform/Uno.ColorCode-Universal](https://github.com/unoplatform/Uno.ColorCode-Universal)
> [unoplatform/Uno.GettingStartedTutorial](https://github.com/unoplatform/Uno.GettingStartedTutorial)
> [unoplatform/Uno.LottieSample](https://github.com/unoplatform/Uno.LottieSample)
> [unoplatform/uno.mvvmlight](https://github.com/unoplatform/uno.mvvmlight)
> [unoplatform/Uno.Prism](https://github.com/unoplatform/Uno.Prism)
> [unoplatform/Uno.QuickStart](https://github.com/unoplatform/Uno.QuickStart)
> [unoplatform/Uno.ReactiveUI](https://github.com/unoplatform/Uno.ReactiveUI)
> [unoplatform/Uno.RoslynQuoter](https://github.com/unoplatform/Uno.RoslynQuoter)
> [unoplatform/Uno.Rx.NET](https://github.com/unoplatform/Uno.Rx.NET)
> [unoplatform/Uno.SQLitePCLRaw.Wasm](https://github.com/unoplatform/Uno.SQLitePCLRaw.Wasm)
> [unoplatform/Uno.UI.Toolkit.SL](https://github.com/unoplatform/Uno.UI.Toolkit.SL)
> [unoplatform/Uno.WindowsStateTriggers](https://github.com/unoplatform/Uno.WindowsStateTriggers)
> [unoplatform/Uno.WindowsStateTriggers](https://github.com/unoplatform/Uno.WindowsStateTriggers)
> [unoplatform/Uno.XamlBehaviors](https://github.com/unoplatform/Uno.XamlBehaviors)
> [unoplatform/Uno.Xaml-Controls-Gallery](https://github.com/unoplatform/Uno.Xaml-Controls-Gallery)

Tracking over at https://github.com/unoplatform/uno/issues/1522

**Pull request merged**